tests/e2e/integration/networks.js

```diff
@@ -0,0 +1,104 @@
+
+import { defaultNetworks, networks, DEFAULT_NETWORK } from '../../../src/popup/utils/constants'
+const network = "Testnet";
```

Comment:
I propose to make e2e test files comply with the current linter settings.

Reply:
I will run linter on `networks.js` after merging https://github.com/aeternity/superhero-wallet/pull/83